todo-frontend/src/stores/userStore.js

```diff
@@ -0,0 +1,70 @@
+import userService from '../services/user.service.js';
+
+export default {
+    namespaced: true,
+    state : {
+        token: null,
+        user: null
+    },
+
+    mutations : {
+        LOGIN(state, t){
+            state.token = t;
+        },
+        DELETE(state) {
+            state.token = '';
+            state.user = null;
+        },
+        USER(state, u) {
+            state.user = u;
+        }
+    },
+
+    actions : {
+        login({commit, state}, data){
```

Comment:
ovaj login mi deluje ogromno :smile: 
aj probaj to da malo pojednostavis i rasparcas
custom promise ti ne treba